src/frontend/overlay/src/components/organisms/widgets/Queue2.js

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { Transition, TransitionGroup } from "react-transition-group";
+import styled, { keyframes } from "styled-components";
+import {
+    QUEUE_BASIC_ZINDEX,
+    QUEUE_FEATURED_RUN_ASPECT,
+    QUEUE_ROW_APPEAR_TIME,
+    QUEUE_ROW_FEATURED_RUN_APPEAR_TIME,
+    QUEUE_ROW_FTS_TRANSITION_TIME,
+    QUEUE_ROW_HEIGHT2,
+    QUEUE_ROW_TRANSITION_TIME, WIDGET_LAYOUT_BACKGROUND
+} from "../../../config";
+import { QueueRow2 } from "../../molecules/queue/QueueRow2";
+import { TeamViewHolder } from "../holder/TeamViewHolder";
+import { useWithTimeoutAfterRender } from "../../../utils/hooks/withTimeoutAfterRender";
+
+const MAX_QUEUE_ROWS_COUNT = 20;
+
+const WidgetWrap = styled.div`
+  width: 100%;
+  height: 100%;
+  position: relative;
+  background-color: ${WIDGET_LAYOUT_BACKGROUND};
+  background-repeat: no-repeat;
+  border-radius: 16px;
+`;
+
+const QueueRowWrap = styled.div.attrs(({ bottom, zIndex }) => ({
+    style: {
+        bottom: bottom + "px",
+        zIndex: zIndex,
+    }
+}))`
+  overflow: hidden;
+  width: 100%;
+
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  transition: bottom linear ${({ fts }) => fts ? QUEUE_ROW_FTS_TRANSITION_TIME : QUEUE_ROW_TRANSITION_TIME}ms;
+  animation: ${({ animation }) => animation} ${QUEUE_ROW_APPEAR_TIME}ms linear;
+  animation-fill-mode: forwards;
+  box-sizing: border-box;
+`;
+
+const FeaturedRunQueueRow = styled.div.attrs(({ zIndex, height }) => ({
+    style: {
+        zIndex: zIndex,
+        height: height,
+    }
+}))`
+  width: 100%;
+  overflow: hidden;
+  position: relative;
+  animation: ${({ animation }) => animation} ${QUEUE_ROW_FEATURED_RUN_APPEAR_TIME}ms ease-in-out;
+  animation-fill-mode: forwards;
+`;
+
+const rowExpand = (fullHeight) => keyframes`
+  from {
+    max-height: 0;
+  }
+
+  to {
+    max-height: ${fullHeight}px;
+  }
+`;
+
+const rowContract = (fullHeight) => keyframes`
+  from {
+    max-height: ${fullHeight}px;
+  }
+
+  to {
+    max-height: 0;
+  }
+`;
+
+const contractionStatesFeatured = (fullHeight) => ({
+    entering: {},
+    entered: { animation: rowExpand(fullHeight) },
```

Comment:
This makes no sense to have animation on entered.
It's just styles that will be applied after timeout on Transition component.
If it's an "appear" animation - it's just a delayed animation.